pandas-stubs/errors/__init__.pyi

```diff
@@ -28,30 +25,24 @@ class AbstractMethodError(NotImplementedError):
 class NumbaUtilError(Exception): ...
 class DuplicateLabelError(ValueError): ...
 class InvalidIndexError(Exception): ...
+class DataError(Exception): ...
+class SpecificationError(Exception): ...
+class SettingWithCopyError(ValueError): ...
+class SettingWithCopyWarning(Warning): ...
+class NumExprClobberingError(NameError): ...
+class IndexingError(Exception): ...
+class PyperclipException(RuntimeError): ...
+
+class PyperclipWindowsException(PyperclipException):
+    def __init__(self, message) -> None: ...
```

Comment:
remove the `__init__()` here - shouldn't be instantiated

Reply:
Removed.